internal/command/run.go

```diff
@@ -50,6 +54,7 @@ func init() {
 
 	runCmd.Flags().StringArrayVarP(&overrideParams, "property", "p", nil, "Overrides selected property value")
 
+	runCmd.Flags().BoolVar(&skipValidation, "skip-validation", false, "DEPRECATED: Disables Score file schema validation")
```

Comment:
Is this introduced as `DEPRECATED` since we don't suggest to use it?

Reply:
It's for backward compatibility before we enforce validation everywhere.